.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,255 @@
+          --min-instances 0 \
+          --timeout 300 \
+          --concurrency 80 \
+          --set-env-vars "APP_ENV=production,USE_GOOGLE_CLOUD_STORAGE=true" \
+          --update-env-vars "BUCKET_NAME=price-comparison-app-data" \
+          --update-env-vars "GOOGLE_APPLICATION_CREDENTIALS_JSON=${{ secrets.GOOGLE_APPLICATION_CREDENTIALS_JSON }}"
+        
+        echo "✅ Cloud Run deployment completed"
+
+    - name: Get deployment URL
+      id: get-url
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:31: Double quote to prevent globbing and word splitting

(shellcheck)